clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1127,7 +1127,9 @@ def OpenMP : DiagGroup<"openmp", [
   ]>;
 
 // SYCL warnings
-def SyclStrict : DiagGroup<"sycl-strict">;
+def Sycl2017Conform : DiagGroup<"sycl-2017-conform">;
```

Comment:
perhaps sycl-2017-compat to be more consistent with c++98-compat, et al?

Reply:
yes, good call.  Done.